### PR TITLE
Update gosigar package after fix

### PR DESCRIPTION
**Pull request by narph**
## What does this PR do?

Described here https://github.com/elastic/gosigar/pull/159

## Why is it important?

Described here https://github.com/elastic/gosigar/pull/159

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Related issues

- Closes https://github.com/elastic/beats/issues/22501

### CODE REVIEW

**Review by jsoriano — APPROVED** [review]

File: .ci/packaging.groovy [hunk 1]
```diff
@@ -429,13 +429,11 @@ def triggerE2ETests(String suite) {
     booleanParam(name: 'notifyOnGreenBuilds', value: !isPR()),
     booleanParam(name: 'BEATS_USE_CI_SNAPSHOTS', value: true),
     string(name: 'runTestsSuites', value: suite),
+    string(name: 'BEAT_VERSION', value: env.BEAT_VERSION),
     string(name: 'GITHUB_CHECK_NAME', value: env.GITHUB_CHECK_E2E_TESTS_NAME),
     string(name: 'GITHUB_CHECK_REPO', value: env.REPO),
     string(name: 'GITHUB_CHECK_SHA1', value: env.GIT_BASE_COMMIT),
   ]
-  if (isPR()) {
```
Comment (kuisathaverat) [inline .ci/packaging.groovy:436]: I think this change is unintencional.

Comment (kuisathaverat) [reply .ci/packaging.groovy:436]: forget it is the same code it is in the master, I dunno why this appears as a change

https://github.com/elastic/beats/blob/master/.ci/packaging.groovy#L420-L442

Comment (narph) [reply .ci/packaging.groovy]: same here, but it is fixed now

**Comment by mdelapenya** [comment]
The failing tests are mostly failing because of a timeout, which is sometimes causing flakiness in the E2E. As I can see those test are not failing for the majority of the other builds, and because the changes in this PR are totally unrelated to the failures, with this root cause analysis in place, I’d say this PR is good to go

**Review by mdelapenya — APPROVED** [review]

**Comment by nathanrstacey** [comment]
A customer showed me the below logs from their 7.12 build. This does not appear to be fixed in 7.12 as this was the same log from 7.11
7_12mectricbeatlog

**Pull request merged**